index.html

```diff
@@ -1,21 +1,139 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <!-- Add a link to your custom CSS here -->
-
-    <title>Responsive grid project</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css"
+        integrity="sha512-9usAa10IRO0HhonpyAIVpjrylPvoDwiPUiKdWk5t3PyolY1cOd4DSE0Ga+ri4AuTroPR5aQvXU9xC6qOPnzFeg=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="/css/main.css">
+    <title>DESIGN 2: Resume(Responsive grid project)</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+    <div class="container">
+        <header>
+            <div class="header-headline">
+                <h1>
+                    <span> <i class="fa-solid fa-square"></i></span> Maya Nelson
+                </h1>
+                <p>
+                    <span class="forward-slash">&#47 </span>
+                    PROJECT MANAGER
+                </p>
+            </div>
+
+            <div class="nav-button">
+                <button class="nav-toggle" id="navToggle">
+                    <i class='fa-solid fa-bars-staggered'></i>
+                </button>
+            </div>
+            <nav>
+                <ul>
+                    <li <a href=""></a> ABOUT ME </li>
+                    <li>RESUME </li>
```

Comment:
as you've created a page for resume, you could link to it here using an `<a>` tag :)